version.tf

```
@@ -0,0 +1,7 @@
+terraform {
+  required_version = ">= 0.13.0, < 0.14"
```

Why not include `>= 0.12.0` ?

```suggestion
  required_version = ">= 0.12.0, < 0.14"
```